src/app/machines/views/MachineDetails/MachineNetwork/AddAliasOrVlan/AddAliasOrVlanFields/AddAliasOrVlanFields.test.tsx

```diff
@@ -1,16 +1,14 @@
-import { mount } from "enzyme";
+import { screen } from "@testing-library/react";
 import { Formik } from "formik";
-import { Provider } from "react-redux";
-import { MemoryRouter } from "react-router-dom";
-import configureStore from "redux-mock-store";
 
 import AddAliasOrVlanFields from "./AddAliasOrVlanFields";
 
 import type { RootState } from "app/store/root/types";
 import { NetworkInterfaceTypes } from "app/store/types/enum";
 import { rootState as rootStateFactory } from "testing/factories";
+import { renderWithBrowserRouter } from "testing/utils";
 
-const mockStore = configureStore();
+const route = "/machines";
```

Comment:
Just noticed one more thing. 
We have a constant for this which you can import:
`import urls from "app/base/urls";`
```suggestion
const route = urls.machines.index;
```

Reply:
Can't believe I forgot about that! Changed it now